pkg/yang/types_builtin.go

```diff
@@ -40,14 +40,17 @@ var (
 	Uint32Range = mustParseRangesInt("0..4294967295")
 	Uint64Range = mustParseRangesInt("0..18446744073709551615")
 
-	Decimal64Range = mustParseRangesDecimal("min..max", 1)
+	Decimal64Range = mustParseRangesDecimal("-922337203685477580.8..922337203685477580.7", 1)
```

Comment:
Given that the range for `decimal64` differs based on the precision, is there something that we need to do to actually check the range within the precision that is specified? I suspect the answer here is yes, and that we rather need to make the maximum values dependent on the fraction digits -- one of the unit tests expects that this range can be used with `fraction-digits 2` which isn't allowed.

Reply:
Thanks for catching this! I almost introduced a new bug. I missed that `fraction-digit` is a required statement, so if we can't use min/max for the base type, then we must populate the `Range` field once we know its value. I've done that now, PTAL.